hibernate-core/src/main/java/org/hibernate/bytecode/enhance/internal/bytebuddy/FieldReaderAppender.java

```diff
@@ -24,19 +24,19 @@
 
```

Comment:
Commit message: I think you should mention the `$$_hibernate_read_<property>()` method, not the `$$_hibernate_read_entity()` method, since `entity` just happens to be the name of the property in the test case.
I might be wrong, but if so I'd like to know what calls this method exactly...